lib/tapioca.rb

```diff
@@ -31,4 +28,12 @@ class Error < StandardError; end
   nil
 end
 
-loader.eager_load
+require_relative "tapioca/cli"
+require_relative "tapioca/loader"
+require_relative "tapioca/compilers/symbol_table/symbol_generator"
+require_relative "tapioca/compilers/symbol_table/symbol_loader"
+require_relative "tapioca/compilers/symbol_table_compiler"
+require_relative "tapioca/constant_locator"
+require_relative "tapioca/gemfile"
+require_relative "tapioca/loader"
+require_relative "tapioca/version"
```

Comment:
I think this order would be more correct:
```ruby
require_relative "tapioca/loader"
require_relative "tapioca/constant_locator"
require_relative "tapioca/generator"
require_relative "tapioca/cli"
require_relative "tapioca/gemfile"
require_relative "tapioca/compilers/symbol_table_compiler"
require_relative "tapioca/compilers/symbol_table/symbol_generator"
require_relative "tapioca/compilers/symbol_table/symbol_loader"
require_relative "tapioca/version"
```

Reply:
Fixed